Makefile

```diff
@@ -30,7 +30,7 @@ E2E_PROVIDER_IMAGE_NAME ?= e2e-provider
 # Release version is the current supported release for the driver
 # Update this version when the helm chart is being updated for release
 RELEASE_VERSION := v1.2.3
-IMAGE_VERSION ?= v1.2.3
+IMAGE_VERSION ?= v1.2.3.0
```

Comment:
Why are we adding an additional 0 at the end?

Reply:
Any change to `docker/` dir in release branch will trigger a new build. We don't want to rebuild `v1.2.3` release again, so this is dummy value.